### DISCUSSION TITLE
✅ 🔗 Is it possible to create a custom widget in which other (nested) can be placed? (Duplicate)

### DISCUSSION THREAD

**Discussion by jeffreyvr** [elementor/elementor#17416] [Feature Request]
Hi,

I'm creating some custom widgets, and for one particular widget I'm looking to be able to add other widgets inside it. So a bit like how the Column and Inner Section works.

Hierarchy:

- Column
  - My custom widget
    - Other widget(s)

I was wondering if this is possible and if someone can give any pointers in how to implement this. I've been looking a bit but can't find any examples to achieve this.

**Comment by m1ga** [comment]
not yet but there is a open (active) PR https://github.com/elementor/elementor/pull/16252
an issue is here: https://github.com/elementor/elementor/issues/13125

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new Nested Elements in Elementor versions 3.8 through 3.15 and more! 🥳 

You can now use:

- The **Loop Builder** - To design your Posts Query
- The **Nested Tabs** - To add any content to your tabs
- The **Nested Carousel** and **Loop Carousel** - To showcase your content in a flexible slider
- The **Mega Menu** - To create bespoke Mega Menus with any widget
- The **Nested Accordion** - Add anything to your accordions

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
